physics/GWD/unified_ugwp.F90

```diff
@@ -488,7 +494,26 @@ subroutine unified_ugwp_run(me, master, im, levs, ak,bk, ntrac, dtp, fhzero, kdt
     ! Note:  In case of GSL drag_suite, this includes ss and tofd
 
     if ( do_gsl_drag_ls_bl.or.do_gsl_drag_ss.or.do_gsl_drag_tofd ) then
-
+!
+if (do_gwd_opt_psl) then
```

Comment:
This should be indented over for clarity.

Reply:
Thanks